packages/components/src/components/dropdown/dropdown.scss

```diff
@@ -269,6 +273,11 @@
 
             .dc-dropdown__display-placeholder-text {
                 color: var(--text-less-prominent);
+                white-space: nowrap;
+                text-overflow: ellipsis;
+                overflow: hidden;
+                display: inline-block;
+                width: 37rem;
```

Comment:
Is this exact value required? Could we replace it with something dynamic like `fit-content` or `100%`?

Reply:
I changed this note to max-width. It allows styles to look dynamic in 2 different types of windows: in settings of account - financial assessment and in the pop-up window "Add a Deriv Financial account”. I didn’t use fit-content or 100% because these properties allow long placeholder overlaps with the dropdown arrow mark. Container in main styles of the app has width: 45%. The app has 2 different types of placeholder: on top input and on the middle input. They are a lot of where and have the same style. That's why suppose, better no-touch main styles.

Reply:
The current solution looks better, but it doesn't make use of ellipsis since the width of the input grows:
![image](https://user-images.githubusercontent.com/71861519/129651452-1459878e-238b-4441-8065-49c6b1d8fe3c.png) I'm not sure if this is what @matin-binary wanted to have?

Reply:
The bug does not say anything about the settings section, so I did not make an ellipsis here, but if necessary, I will do it here as well as in the modal window